src/app/launcher/model/mission.model.ts

```diff
@@ -1,7 +1,9 @@
 export class Mission {
-  missionId: string;
-  suggested?: boolean;
-  title: string;
-  description: string;
-  supportedRuntimes: string[];
+  constructor(
+    public id: string,
+    public name: string,
+    public suggested: boolean,
+    public runtimes: string[],
+    public description: string = null
+  ) {}
```

Comment:
Looking at the demo service, it's not clear why is there a constructor for Mission, but not the Runtime model?